server/src/models/QwixxBaseGameCard.ts

```diff
@@ -16,26 +16,29 @@ export default class qwixxBaseGameCard {
 
     this._numbers = Array.from({ length: 12 }, (_, i) => i + 2);
 
-    this._lockedRows = [];
+    this._isLocked = {
+      [rowColour.Red]: false,
+      [rowColour.Yellow]: false,
+      [rowColour.Green]: false,
+      [rowColour.Blue]: false,
+    };
 
     this._penalties = [];
   }
 
   serialize() {
     return {
-      gameCard: this._rows,
-      numbers: this._numbers,
-      lockedRows: this._lockedRows,
-      penalties: this._penalties,
+      rows: this._rows,
+      isLocked: this._isLocked,
+      penalties: this._penalties.length,
     };
   }
 
   static from(data: any): qwixxBaseGameCard {
     const gameCard = new qwixxBaseGameCard();
-    gameCard._rows = data.gameCard;
-    gameCard._numbers = data.numbers;
-    gameCard._lockedRows = data.lockedRows;
-    gameCard._penalties = data.penalties;
+    gameCard._rows = data.rows;
+    gameCard._isLocked = data.isLocked;
+    gameCard._penalties = new Array(data.penalties).fill(1);
     return gameCard;
   }
```

Comment:
My understanding of this "static from()" method is that it would create a new game card from possibly JSON data. I'm not sure if we need it for now. I imagine a possible use case for it would be if a game was stopped and returned to later. The game data could be saved to a database. Is that what you were thinking? Or is it for something else?

Reply:
Yeah to be honest this might be overkill. I will comment out for now. I dont think it is necessary so I will actually delete it.